Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -28,8 +28,9 @@ public static class UwpAppxBuildTools
         /// </summary>
         /// <param name="buildInfo"></param>
         /// <param name="cancellationToken"></param>
+        /// <param name="buildOnMultipleCores"></param>
         /// <returns>True, if the appx build was successful.</returns>
-        public static async Task<bool> BuildAppxAsync(UwpBuildInfo buildInfo, CancellationToken cancellationToken = default)
+        public static async Task<bool> BuildAppxAsync(UwpBuildInfo buildInfo, CancellationToken cancellationToken = default, bool buildOnMultipleCores = false)
```

Comment:
Is there any reason why we shouldn't enable this for the UWP build in UwpPlayerBuildTools.BuildPlayer?

Reply:
As far as I can tell there is no reason not to enable it.

I tried to designe it with a minimal amount of possible conflicts in mind.


OR 
If you mean rather you would like to always use multicore builds, I'd say that it might be that in some cases, people would like to build in background on only one core.


To be honest I'm not sure what your question really is about, maybe you could elaborate it a little more. 😅

Reply:
Hey, sorry. I guess it would be nice if we can enable this from the command line build (see `UwpPlayerBuildTools.ParseBuildCommandLine`) and also change our yaml templates so that we provide this flag during ci builds (see `pipelines\`). We can try it out during a PR build then.

Reply:
If you need a hand with the yaml files in `pipelines\` I can give it a look instead.

Reply:
I made a new commit, with a merge from current dev.
There I changed the multicore switch to a commandline argument and a corresponding UWPBuildInfo attribute.
It should work as UnityArgs, with the pipeline I guess.